recipes/boost/all/conanfile.py

```diff
@@ -129,6 +132,32 @@ class BoostConan(ConanFile):
     def export(self):
         self.copy(self._dependency_filename, src="dependencies", dst="dependencies")
 
+    @property
+    def _min_compiler_version_default_cxx11(self):
+        # Minimum compiler version having c++ standard >= 11
+        return {
+            "gcc": 6,
+            "clang": 6,
+            "apple-clang": 12,  # guess
+            "Visual Studio": 14,  # guess
+        }.get(str(self.settings.compiler))
+
+    @property
+    def _min_compiler_version_nowide(self):
+        # Nowide needs c++11 + swappable std::fstream
+        return {
+            "gcc": 5,
+            "clang": 5,
+            "apple-clang": 12,  # guess
+            "Visual Studio": 14,  # guess
+        }.get(str(self.settings.compiler))
+
+    @property
+    def _fiber_minimum_compiler_version(self):
+        return {
+            "clang": "6",
+        }.get(str(self.settings.compiler))
+
```

Comment:
```suggestion
```

Don't see the usage, and having this for clang only is confusing.
Probably should be deleted.